craft_archives/repo/apt_key_manager.py

```diff
@@ -229,3 +243,15 @@ def install_package_repository_key(
             self.install_key_from_keyserver(key_id=key_id, key_server=key_server)
 
         return True
+
+
+def _gpg_prefix() -> List[str]:
+    """Create a gpg command line that includes options that we always want to use."""
+    return ["gpg", "--batch", "--no-default-keyring"]
+
+
+def _gnupg_ring(keyring_file: pathlib.Path) -> str:
```

Comment:
This seems like it could be a `@property` on the `AptKeyManager`

Reply:
yes but then the linters complain that it could be a free function... somebody's going to end up unhappy! hehe